sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -38,80 +47,116 @@ pub struct InputSelection {
     required_inputs: HashSet<OutputId>,
     forbidden_inputs: HashSet<OutputId>,
     selected_inputs: Vec<InputSigningData>,
-    outputs: Vec<Output>,
+    context_inputs: HashSet<ContextInput>,
+    provided_outputs: Vec<Output>,
+    added_outputs: Vec<Output>,
```

Comment:
Are `automatically_transitioned` also added to this ?

Reply:
yes

Reply:
Do we still need `automatically_transitioned`?

Reply:
possibly not, but I tried to remove it and ran into some weirdness so I don't want to touch it

Reply:
A topic for later